experiment/test_dispatcher.py

```diff
@@ -49,8 +50,15 @@ def mock_split_successes_and_failures(inputs, results):
 @mock.patch('multiprocessing.pool.ThreadPool', test_utils.MockPool)
 @mock.patch('experiment.build.builder.split_successes_and_failures',
             mock_split_successes_and_failures)
-def dispatcher_experiment(fs, db, experiment):
+def dispatcher_experiment(fs, db, environ):
     """Creates a dispatcher.Experiment object."""
+    work = '/work'
+    os.environ['WORK'] = work
+    fs.create_dir(work)
+    os.environ['EXPERIMENT'] = 'test-experiment'
```

Comment:
Why are we not using experiment helper that sets all these vars.

Reply:
Done.